Gemfile.lock

```diff
@@ -1,96 +1,98 @@
+PATH
+  remote: .
+  specs:
+    art19-url_validation (1.1.2)
+      activerecord (>= 5.1.7)
+      activesupport (>= 5.1.7)
+      addressable (~> 2.6.0)
+      httpi (~> 2.4.4)
+
 GEM
   remote: https://rubygems.org/
   specs:
-    RedCloth (4.2.9)
-    activemodel (4.1.7)
-      activesupport (= 4.1.7)
-      builder (~> 3.1)
-    activerecord (4.1.7)
-      activemodel (= 4.1.7)
-      activesupport (= 4.1.7)
-      arel (~> 5.0.0)
-    activesupport (4.1.7)
-      i18n (~> 0.6, >= 0.6.9)
-      json (~> 1.7, >= 1.7.7)
+    RedCloth (4.3.2)
+    activemodel (6.0.2.2)
+      activesupport (= 6.0.2.2)
+    activerecord (6.0.2.2)
+      activemodel (= 6.0.2.2)
+      activesupport (= 6.0.2.2)
+    activesupport (6.0.2.2)
+      concurrent-ruby (~> 1.0, >= 1.0.2)
+      i18n (>= 0.7, < 2)
```

Comment:
I think you need to limit the `activemodel` to the consisten `5.1.7` constraint. otherwise you bring in 6+ which yields quite some warning.